### ISSUE TITLE
Pdfkit import fails in Electron

### ISSUE THREAD

**Issue by borremosch** [electron/electron#24163]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * v9.0.4
* **Operating System:**
  * Windows 10 (2004)

### Expected Behavior
I expect my project to compile and run when I import pdfkit

### Actual Behavior
The project compiles fine. However, it crashes on runtime with a very cryptic error message.

### To Reproduce
`import PDFDocument from 'pdfkit';`

### Additional Information
The error message is:

```Uncaught Error: Invalid package C:\Users\me\project\node_modules\electron\dist\resources\electron.asar
    at createError (electron/js2c/asar.js:126)
    at Object.fs.readFileSync (electron/js2c/asar.js:545)
    at Object../node_modules/fontkit/index.js (ArabicShaper.js:5)
    at __webpack_require__ (bootstrap:789)
    at fn (bootstrap:100)
    at Module../node_modules/pdfkit/js/pdfkit.es5.js (index.js:20)
    at __webpack_require__ (bootstrap:789)
    at fn (bootstrap:100)
    at Module../src/renderer/lib/exportToPdf.ts (exportToPdf.ts:3)
    at __webpack_require__ (bootstrap:789)
```

**Comment by codebytere** [comment]
@borremosch i ran https://gist.github.com/28e2ac05736db067aec8d031a798b9c6 and experienced no issues - without a reproducible case we can't triage this. Please add a repro that demonstrates this failing as a result of Electron itself (as opposed to within a larger complex app) in order for this issue to remain open - it will be closed in 7 days if more information is not provided.

**Comment by borremosch** [comment]
@codebytere thank you for creating the gist, it has put me on the right track in finding the problem. I was running a development build, and when I tried a production build, my Electron project outputted a more useful error message, which indicated a file was missing during runtime.

I was able to fix this problem by using transformer-loader in my Webpack config. I'll just leave the related pdfkit issue here in case somebody with the same problem comes across this issue: https://github.com/foliojs/fontkit/issues/67.

In hindsight, I am still not sure as to the role Electron plays in all of this. I guess from the Electron side, it would have been helpful to get a more descriptive error message. But other than that, it seems that a specific combination of pdfkit + Electron + Webkit caused the issue.
